apps/sms/views/conversation/js/conversation.js

```diff
@@ -763,15 +767,14 @@ var ConversationView = {
           }
 
           this.recipients.add(recipient);
-        }.bind(this));
-      }, this);
+
+          this.draft.isEdited = false;
+        });
+      });
 
       // Render draft contents into the composer input area.
       Compose.fromDraft(this.draft);
 
-      // Discard this draft object and update the backing store
-      Drafts.delete(this.draft).store();
```

Comment:
yeah I agree this should be a feature of dialer (we actually nearly did the feature in the past) but this can be a workaround by the user.